### PR TITLE
pull

### PR DESCRIPTION
**Pull request by PrettyPearl2302**
## Description

I wrote a function taddyGraphqlRequest to pull data from my external Taddy API. I also updated my data models in models.js to match with the data i will be getting from the API. 

## Milestones
podcast display page and home page

## Resources
[taddy example project](https://github.com/taddyorg/example-project/blob/main/index.js)
[graphql documentation](https://graphql.org/learn/)

## Test Plan
i called the function to test it at the end of my app.js file. Taddy identifies podcasts by uuids, so i tested my function with the uuid of a specific podcast. 
Screenshot 2023-07-11 at 12 04 32 PM
Screenshot 2023-07-11 at 12 04 56 PM

### CODE REVIEW

**Comment by jeffwdeng** [comment]
We should update this pull request name to a more descriptive title

File: timify-api/app.js [hunk 1]
```diff
@@ -1,37 +1,134 @@
...
+      podcastSeries{
+        uuid
+        name
+        rssUrl
+        itunesId
+      }
+    }
+  }`;
+
+  // module.exports = {
```
Comment (jeffwdeng) [inline timify-api/app.js]: delete the comments

File: timify-api/app.js [hunk 2]
```diff
@@ -1,37 +1,98 @@
...
+    console.log("inside sentTaddyGraphqlRequest", e)
+  }
 
-app.get('/', (req, res) => {
-  res.send("it works"); 
-});
+  }
 
-// Route to get all podcasts
-app.get('/podcasts', async (req, res) => {
```
Comment (ssorenson130) [inline timify-api/app.js:19]: why are we deleting this?

File: timify-api/app.js [hunk 3]
```diff
@@ -1,37 +1,98 @@
...
+        uuid
+        publisherId
+        publisherName
+        baseArtworkUrl
+        baseArtworkUrlOf(size: 640)
+      }
     }
-  });
-
-sequelize.sync({ alter: true })
```
Comment (ssorenson130) [inline timify-api/app.js:28]: same here. do we need to remove this?

File: timify-api/models/podcast.js [hunk 4]
```diff
@@ -1,25 +1,55 @@
...
+  },
+  rssURL: {
+    type: DataTypes.STRING,
+    allowNull: false
+  },
+  rssOwnerName: {
+    type: DataTypes.STRING,
+    allowNull: false
+  },
+  rssOwnerPublicEmail: {
```
Comment (ssorenson130) [inline timify-api/models/podcast.js:47]: i see we're changing the podcast data model to match the output of the api. do we need _all_ of these fields though?

**Comment by ghost** [comment]
For test plan, could you add some details how you test that API function and what the returned value looks like? After you enter uuid, what result we will get from it? Would be better if you can attach that result in test plan.

**Pull request merged**